Wiki/models/models-aus2200.md

```diff
@@ -0,0 +1,171 @@
+```
+The `create_soil_moisture_ancil` task is configured to wait for the completion of the `aus2200_d0198_RA3_um_fcst_000` task from the previous suite. Therefore, it is possible to run multiple versions of the AUS2200 suite that rely on soil moisture carryover simultaneously. This will allow initial and boundary conditions to be created and cached ahead of time, whilst the reconfiguration and model run tasks will only commence when the data required is present. This saves a significant amount of resources, as it is typical for model runs to overlap due to the required spinup time. Later model runs can therefore use cache boundary conditions created for earlier cases.
+
+### Typical restart workflow
+
+A typical daily restart run will require multiple copies of the AUS2200 suite to run simultaneously. The following encapsulates the workflow for the AUS2200 model run described [here](https://forum.access-hive.org.au/t/experiment-proposal-simulating-east-coast-low-complex-in-early-june-2016-using-aus2200/1507). The simulations took place over 2-9 June 2016 with a 2-day soil moisture spinup preceding the full simulation. The simulation will be restarted every day, and run with a 24 hour spin-up time. This means that a total of 9 variants of the AUS2200 suite will be run.
+
+![simulation-diag](../Images/AUS2200/simulation-times.png)
+
+Therefore, the full model run needs to start on the 30th of May 2016. For this initial simulation, `SOIL_MOISTURE_CARRYOVER` must be false, item `1` in the 'Configure ancils and initialise dump fields' section of the `um` task must be set to ignored, `INITIAL_CYCLE_POINT` must be set to 20160530T0000Z, and `FINAL_CYCLE_POINT` must be set to 20160503T1800Z. Submit this suite with `rose suite-run`.
+
+In your `~/roses` directory, make a copy of the AUS2200 suite, following the naming convention outlined [above](#restart-and-spinup). This example uses `u-cs142` for the original suite id.
+<terminal-window lineDelay=0>
+    <terminal-line data="output"><span style="color:#A6CE39">u-cs142</span><span style="color:#FAA619">\$&nbsp;</span>cd ..</terminal-line>
+    <terminal-line data="output"><span style="color:#A6CE39">roses</span><span style="color:#FAA619">\$&nbsp;</span>cp -r u-cs142 u-cs142-20160601T0000</terminal-line>
+    <terminal-line data="output"><span style="color:#A6CE39">roses</span><span style="color:#FAA619">\$&nbsp;</span>cd u-cs142-20160601T0000</terminal-line>
+    <terminal-line data="output"><span style="color:#A6CE39">u-cs142-20160601T0000</span><span style="color:#FAA619">\$&nbsp;</span>rose edit</terminal-line>
+</terminal-window>
+With the rose config editor window open, change `INITIAL_CYCLE_POINT` to match the date stamp of the directory, set `FINAL_CYCLE_POINT` to 20160602T1800. Re-enable `SOIL_MOISTURE_CARRYOVER` and ensure `SPINUP_HOURS` is set to 24. Ensure that item `1` in the 'Configure ancils and initialise dump fields' section of the `um` task has been re-enabled. 
```

Comment:
should um be capitalised?

Reply:
No, this is the name of the section in the rose config editor, 'um' is lower case.